apps/explorer-e2e/src/integration/asset.cy.js

```diff
@@ -0,0 +1,20 @@
+import '../support/common.functions';
+
+context('Asset page', function () {
+  describe('Verify elements on page', function () {
+    const assetsNavigation = 'a[href="/assets"]';
+    const assetHeader = '[data-testid="asset-header"]';
```

Comment:
comment (non-blocking) Don't know if it makes any change, but consts may be declared even before context block